neqo-transport/src/connection.rs

```diff
@@ -155,6 +153,20 @@ impl Output {
     }
 }
 
+/// Used by inner functions like Connection::output.
+enum SendOption {
+    /// Yes, please send this datagram.
+    Yes(Datagram),
+    /// Don't send.  If this was blocked on the pacer (the arg is true).
+    No(bool),
+}
+
+impl Default for SendOption {
```

Comment:
Would this be better as a named function instead, like `no()`?

Reply:
I think it is fine as it is.